templates/.github/workflows/go-check.yml

```diff
@@ -16,6 +18,12 @@ jobs:
       - name: Run repo-specific setup
         uses: ./.github/actions/go-check-setup
         if: hashFiles('./.github/actions/go-check-setup') != ''
+      - name: Read config
+        if: hashFiles('./.github/workflows/go-check-config.json') != ''
+        run: |
+          if [[ $(jq -r .gogenerate ./.github/workflows/go-check-config.json) == "true" ]]; then
```

Comment:
nit: use the `-e` flag in jq instead of parsing the output? (

Reply:
Cool, I wasn't aware of that flag!